src/devsecops/cli/services/nexus.py

```diff
@@ -180,3 +180,49 @@ def dso_nexus_update_group_repo(url, login_username, login_password, verbose,
         sys.stderr.write(f'Error updating group repo {repo}:\n{error}\n')
     sys.stderr.flush
     exit(exit_code)
+
+@dso_nexus.command(name='add-script')
+@opts.default_opts
+@click.option('--script-name', '-n', required=True,
+              help=('the name of the script to add'))
+@click.option('--script-content', '-c', required=True,
+              help=('the script\'s content'))
+@click.option('--script-type', '-t', required=True,
+              help=('the script\'s type'))
+def dso_nexus_add_script(url, login_username, login_password, verbose,
+                       script_name, script_content, script_type):
+    """Add new Scripts to the Nexus instance specified by URL"""
+    exit_code = 0
+    with nexus.Nexus(
+        url, login_username, login_password, verbosity=verbose
+    ) as api:
+        try:
+            api.add_script(script_name, script_content, script_type)
+        except Exception as e:
+                exit_code += 1
+                print(f'{script_name} failed')
+
+    sys.stderr.flush
```

Comment:
If you're just using `print` it's synchronous, and it's on `sys.stdout` anyways. You can ditch the `flush`, and you should be able to simply `exit(1)` or `raise e` or however you like to fail here since it's not looping. Means 195 can go, too.